Formula/opusfile.rb

```diff
@@ -1,7 +1,7 @@
 class Opusfile < Formula
   desc "API for decoding and seeking in .opus files"
   homepage "https://www.opus-codec.org/"
-  url "https://downloads.xiph.org/releases/opus/opusfile-0.12.tar.gz"
+  url "https://ftp.osuosl.org/pub/xiph/releases/opus/opusfile-0.12.tar.gz"
```

Comment:
Would a GitLab tarball here be better? https://gitlab.xiph.org/xiph/opusfile/-/archive/v0.12/opusfile-v0.12.tar.bz2

It might not have the `configure` script though.

Reply:
Given https://github.com/Homebrew/homebrew-core/pull/78815#issuecomment-855310197 I think the `ftp.osuosl.org` URL would be fine, but I don't feel too strongly about this.

Reply:
Ah, yes, missed that one. This is fine.